dockerfiles/django/entrypoint.sh

```diff
@@ -6,10 +6,10 @@ cmd="$@"
 function postgres_ready(){
 python << END
 import sys
-import psycopg2
+import psycopg
 try:
-    conn = psycopg2.connect(dbname="$POSTGRES_DB", user="$POSTGRES_USER", password="$POSTGRES_PASSWORD", host="$POSTGRES_HOST", port="$POSTGRES_PORT")
-except psycopg2.OperationalError:
+    conn = psycopg.connect(dbname="$POSTGRES_DB", user="$POSTGRES_USER", password="$POSTGRES_PASSWORD", host="$POSTGRES_HOST", port="$POSTGRES_PORT")
```

Comment:
Here you could have used the `DATABASE_URL` variable.
https://www.psycopg.org/psycopg3/docs/api/connections.html#psycopg.Connection.connect

Reply:
We should remove this altogether. I just haven't gotten around to it yet. Thank you for bringing it up.